frontend/public/index.html

```diff
@@ -1,6 +1,14 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <meta
+        http-equiv="Content-Security-Policy"
+        content="
+            default-src 'self';
+            script-src 'self' dap.digitalgov.gov www.google-analytics.com;
+            style-src 'self' 'unsafe-inline';
+            connect-src 'self' www.google-analytics.com %REACT_APP_BACKEND_URL%;
```

Comment:
I must have missed the requests to `https://dc.services.visualstudio.com/v2/track` when writing the original ticket because they're blocked by a browser extension, but I'm guessing we want to allow those since [the app is manually sending those reports to Azure application insights](https://github.com/CDCgov/prime-simplereport/blob/main/frontend/src/app/AppInsights.js).

Should we just add dc.services.visualstudio.com to the `connect-src` clause?

Reply:
Yes we should add `dc.services.visualstudio.com` to `connect-src`

Reply:
Added in 5f9b44c. The domain is no longer blocked by the security policy. I still get an error in the browser on requests to that domain (see pic), but I think I'm just missing a bit of required configuration.

<img width="435" alt="Screen Shot 2021-03-24 at 11 49 42 AM" src="https://user-images.githubusercontent.com/27730981/112340741-390c7480-8c97-11eb-9d14-04477cc81117.png">